File: Python/conditionals.py

"""
conditionals.py
Asks the user for a number and checks if it is even or odd.
"""

def check_even_odd():
    number = int(input("Enter a number: "))
    if number % 2 == 0:
        print("The number is even.")
    else:
        print("The number is odd.")

check_even_odd()

File: Java/Conditionals.java

/*
Conditionals.java
Checks if a given number is even or odd.
*/

import java.util.Scanner;

public class Conditionals {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int number = scanner.nextInt();
        if (number % 2 == 0) {
            System.out.println("The number is even.");
        } else {
            System.out.println("The number is odd.");
        }
        scanner.close();
    }
}
